meshpy/utility.py

```diff
@@ -328,3 +328,23 @@ def get_min_max_nodes(nodes, *, middle_nodes=False):
                         min_max_nodes.append(node_list[index])
                 geometry[f"{direction}_{text}"] = GeometrySet(min_max_nodes)
     return geometry
+
+
+def find_node_on_plane(node, *, normal=None, origin_distance=None, tol=mpy.eps_pos):
```

Comment:
Maybe change the name to `is_node_on_plane` or  `query_node_on_plane`? Since this function does not actually search/find anything.

Also, lets add an second optional parameter `point_on_plane` (mutually exclusive with `origin_distance`) where one can give a point on the plane to define it, I assume in some cases it can be beneficial yo have this option as well.

Reply:
Added the second option in the latest commit.